Remaining section review (#11)
ezsystems/commerce-documentation: docs/api/rest_api.md

# REST API

eZ Commerce extends the REST API of eZ Platform with the following features:

## Basket API

| Feature | Method | Notes |
| ------- | ------ | ----- |

> |Get current basket	|`/api/ezp/v2/siso-rest/basket/current`||

Reviewer: ```suggestion
|Gets current basket	|`/api/ezp/v2/siso-rest/basket/current`||
```

|Reads list of baskets|	`/api/ezp/v2/siso-rest/basket/header/{basket_type}`|Returns wishlist/stored baskets|
|Updates party information in the basket|`/api/ezp/v2/siso-rest/basket/current/party/invoice (PATCH)`</br>`/api/ezp/v2/siso-rest/basket/current/party/delivery (PATCH)`</br>`/api/ezp/v2/siso-rest/basket/current/party/buyer (PATCH)`||

> |Update shipping infos|	`/api/ezp/v2/siso-rest/basket/current/shippingmethod` ||

Reviewer: ```suggestion
|Updates shipping information|	`/api/ezp/v2/siso-rest/basket/current/shippingmethod` ||
```

> |Update payment infos|	`/api/ezp/v2/siso-rest/basket/current/paymentmethod`	||

Reviewer: ```suggestion
|Updates payment information|	`/api/ezp/v2/siso-rest/basket/current/paymentmethod`	||
```

> |Update and check voucher|	`/api/ezp/v2/siso-rest/basket/current/voucher`||

Reviewer: ```suggestion
|Updates and checks voucher|	`/api/ezp/v2/siso-rest/basket/current/voucher`||
```

|Adds a product to a basket|	`/api/ezp/v2/siso-rest/basket/current/lines`||

> |Update infos in the basketline	|`/api/ezp/v2/siso-rest/basket/update` ||

Reviewer: ```suggestion
|Updates information in the basketline	|`/api/ezp/v2/siso-rest/basket/update` ||
```

## Checkout API

| Feature | Method |
| ------- | ------ |

> |Get list of payment methods | `/api/ezp/v2/siso-rest/checkout/payment-methods`	|

Reviewer: ```suggestion
|Gets list of payment methods | `/api/ezp/v2/siso-rest/checkout/payment-methods`	|
```

> |get list of shipping methods | `/api/ezp/v2/siso-rest/checkout/shipping-methods`	|

Reviewer: ```suggestion
|Gets list of shipping methods | `/api/ezp/v2/siso-rest/checkout/shipping-methods`	|
```

## Common API

| Feature | Method |
| ------- | ------ |

> |Get list of payment methods | `/api/ezp/v2/siso-rest/country-selection` |

Reviewer: ```suggestion
|Gets list of payment methods | `/api/ezp/v2/siso-rest/country-selection` |
```

## Customer API

| Feature | Method |
| ------- | ------ |

> |Get list of shipping addresses from customer | `/api/ezp/v2/siso-rest/customer/addresses/shipping`|

Reviewer: ```suggestion
|Gets list of shipping addresses from customer | `/api/ezp/v2/siso-rest/customer/addresses/shipping`|
```
